### ISSUE TITLE
Widget Editor: When `SHIFT + TAB` ing to the top, some problems arise

### ISSUE THREAD

**Issue by mapk** [WordPress/gutenberg#25876]
**Two issues here.**

### 1. "Select parent (widget area) doesn't show a selection
In the screencast below, I was using `SHIFT + TAB` to keyboard back to the top and got to a point where it said "Select parent (widget area)" but didn't show that anything was selected. My guess is that the widget area was selected but it was not highlighted with a blue border. It should be highlighted with a blue border.
 
### 2. Add block above top block
In the screencast below, I was using `SHIFT + TAB` to keyboard back to the top and an Add block `+` appeared which was awkward. I suggest that doesn't appear above the first block in the widget area.

![add-block](https://user-images.githubusercontent.com/617986/95265920-081ead00-07e7-11eb-88b3-a57fbb71d592.gif)

**Comment by talldan** [comment]
Issue 1 should have been solved by https://github.com/WordPress/gutenberg/pull/26011, that button isn't present at all any more.

For Issue 2, this also happens in the post editor. I don't think we'd be able to apply a fix just for the widget editor, Ideally any change works across editors. I think it needs accessibility feedback too before a dev works on it, as it removes one of the ways a block can be added between blocks. It'd probably be quite difficult to add between blocks without this button, the more menu option is possible, but only inserts paragraphs.

BTW, there have been improvements now from the previous screenshots, there was a bug in the widget area CSS that removed some margin that was previously present, now the Add Block button at least shows within the widget area:
<img width="465" alt="Screenshot 2020-10-19 at 5 22 45 pm" src="https://user-images.githubusercontent.com/677833/96426563-f9e17f80-122f-11eb-8579-6fd11f5a6420.png">

**Comment by mapk** [comment]
Thanks for solving issue 1. 

And you're right about issue 2. I never realized that. I'll close this issue because it's definitely not solely related to the Widget Editor. 

![Screen Shot 2020-10-19 at 1 38 52 PM](https://user-images.githubusercontent.com/617986/96509174-7fedce00-1210-11eb-841c-2fc77b178751.png)
